react/src/components/ServiceLauncherPageContent.tsx

```diff
@@ -784,6 +866,17 @@ const ServiceLauncherPageContent: React.FC<ServiceLauncherPageContentProps> = ({
                         // }}
                         />
                         <ResourceAllocationFormItems />
+                        <Form.Item
+                          name={'envvars'}
```

Comment:
This `Form.Item` is only for displaying a label, so it should not have a `name` because EnvVarFormList is a kind of `Form.Item` with a `name`.`name`. 
```suggestion
```